source/mongodump.txt

```diff
@@ -25,18 +25,22 @@
 Synopsis
 --------
 
-:binary:`~bin.mongodump` is a utility for creating a binary export of
-the contents of a database. :binary:`~bin.mongodump` can export data
-from either :binary:`~bin.mongod` or :binary:`~bin.mongos` instances;
-i.e. can export data from standalone, replica set, and sharded cluster
-deployments.
+:binary:`~bin.mongodump` is a utility that creates a binary export of
+a database's contents.
+
+:binary:`~bin.mongodump` can export data from:
+
+- :binary:`~bin.mongod` and :binary:`~bin.mongos` instances.
+- Standalone deployments.
+- Replica sets.
+- Sharded clusters.
 
 .. include:: /includes/extracts/require-cmd-line-mongodump.rst
 
 .. seealso::
 
-   :binary:`~bin.mongorestore` which provides the corresponding
-   binary data import capability.
+   :binary:`~bin.mongorestore`, which allows you to import data that was
```

Comment:
Consider shortening further: 
:binary:`~bin.mongorestore`, to import data that was exported using :binary:`~bin.mongodump`.

Reply:
Prefer to keep as is for clarity.